tests/std/tests/GH_002030_asan_annotate_string/test.cpp

```diff
@@ -1845,6 +1835,13 @@ void test_DevCom_10116361() {
     s1.~string();
 }
 
+void test_DevCom_10109507() {

```

Comment:
We don't seem to test `string::replace` in other areas of this test - perhaps should add thorough tests for that function like the others?